src/ccall.cpp

```diff
@@ -2081,7 +2066,6 @@ jl_cgval_t function_sig_t::emit_a_ccall(
 
     // Mark GC use before **and** after the ccall to make sure the arguments
```

Comment:
There's a few comment that can be removed too. e.g. here and the one before `CreateTrap` below.